packages/typedoc-plugin-appium/test/e2e/converter/comment.e2e.spec.ts

```diff
@@ -8,21 +8,16 @@ import {
 import {initConverter} from '../helpers';
 
 describe('@appium/typedoc-plugin-appium', function () {
-  describe('deriveComment()', function () {
+  describe('comments', function () {
     let knownMethods: KnownMethods;
 
     before(async function () {
       const converter = await initConverter(BuiltinExternalDriverConverter, NAME_TYPES_MODULE);
       knownMethods = converter.convert();
       expect(knownMethods.size).to.be.greaterThan(0);
-      expect(knownMethods.get('activateApp')!.comment).to.exist;
+      expect(knownMethods.get('activateApp')).to.exist;
     });
 
-    describe('when not provided a reflection nor `Comment` parameter', function () {
-      it('should derive the comment from KnownMethods', function () {
-        const commentData = deriveComment('activateApp', knownMethods);
-        expect(commentData).to.exist.and.to.have.keys('comment', 'commentSource');
-      });
-    });
+    it('should', function () {});
```

Comment:
i don't imagine you intend to commit an empty test?

Reply:
I didn't, no.